Doozuu/230620/pro_124나라의숫자.js

```diff
@@ -0,0 +1,11 @@
+// 3으로 나눈거를 다시 3으로 나눈 나머지에 해당하는 값을 계속 앞에 붙이기
+function solution(n) {
+  let answer = '';
+  let arr = [4, 1, 2];
```

Comment:
```suggestion
  let arr = ['4', '1', '2'];
```
요 정수 형식의 데이터가 밑에서 쓰이지 않으니, 처음부터 string 형식으로 저장해두면 좋을 것 같아요!

Reply:
n%3 연산때문에 숫자로 쓴거였는데 string으로 해도 똑같군요..! 감사합니다:)